rosetta-client/src/wallet.rs

```diff
@@ -143,10 +143,10 @@ impl Wallet {
         &self.account
     }
 
-    /// Returns the current block identifier.
+    /// Returns the latest finalized block identifier.
     pub async fn status(&self) -> Result<BlockIdentifier> {
         let status = self.client.network_status(self.config.network()).await?;
-        Ok(status.current_block_identifier)
+        Ok(status.latest_finalized_block_identifier)
```

Comment:
hmm.. who use this method? seems it is being used only in the ethereum examples directory.

Reply:
the timechain uses this method

Reply:
the timechain uses three  functions, `create_wallet`, `status` and `eth_view_call`. we could probably delete 99% of our connector code :slightly_smiling_face: